backend/src/main/java/com/huellapositiva/infrastructure/orm/repository/JpaStatusRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.huellapositiva.infrastructure.orm.repository;
+
+import com.huellapositiva.infrastructure.orm.entities.JpaStatus;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface JpaStatusRepository extends JpaRepository<JpaStatus, Integer> {
+
+    @Query("FROM JpaStatus s WHERE s.id = :id")
+    Optional<JpaStatus> findById(@Param("id") Integer id);
+
+}
```

Comment:
What is this class for? What is JpaStatus, is it JpaProposalStatus? It has a very ambiguous name.

Reply:
Ok Ibai, is fixed the problem. Delete JpaStatus.